x-pack/plugin/security/src/test/java/org/elasticsearch/license/LicensingTests.java

```diff
@@ -304,23 +300,60 @@ private static void assertElasticsearchSecurityException(ThrowingRunnable runnab
         assertThat(ee.status(), is(RestStatus.FORBIDDEN));
     }
 
-    public static void disableLicensing() {
-        disableLicensing(License.OperationMode.BASIC);
-    }
-
-    public static void disableLicensing(License.OperationMode operationMode) {
-        for (XPackLicenseState licenseState : internalCluster().getInstances(XPackLicenseState.class)) {
-            licenseState.update(operationMode, false, null);
-        }
+    private void disableLicensing() throws InterruptedException {
+        // do this in an await busy since there is a chance that the enabled action is not held in place by a node and the node
```

Comment:
I don't follow what `not held in place` means in this comment.

Reply:
You're right, this is not clear. I updated the comment, but essentially I am saying the update performed to the license gets overwritten